### ISSUE TITLE
Cannot find config option to save custom words into dictionary

### ISSUE THREAD

**Issue by jimklimov** [klausw/hackerskeyboard#580]
On my previous phone (Android 4.1.x) with Hacker's Keyboard, when I finished typing a word the band with suggestions displayed an arrow to the typed word (in the band), so I could click and save it. This helped a lot with email addresses, nicknames, IT slang, etc. On the new phone (Android 6) I do not see such a feature, also tried toggling various configurable options with no effect for this (lack of) feature.

Does my memory fail me, or is this user-dictionary support limited by the OS? Other keyboards on Android 6 offer to use the google-backed dictionary for the user account; perhaps HK could do so too?

Side note: stock keyboards suck with multi-language support, only offering to fix English words... or mabe it is just my vendor's ROM problem... HK with downloaded dictionaries rocks :)

**Comment by TPS** [comment]
This still works exactly as it did in previous OS versions, even on 7/Nougat that I'm running:  Type anything _not_ in any dictionary, &, before hitting any word-ending (whitespace/punctuation) key, press white/left-most word in suggestion bar.  Your "add" prompt should appear.

**Comment by jimklimov** [comment]
Unfortunately, for me this just pastes the word into text area (and adds a space after it).
Maybe some incompatibility with Xiaomi MIUI8 then...

**Comment by TPS** [comment]
That's the typical behavior for something _in_ the dictionary.  Did you double-check w/ an obvious nonsense term?  Else, [this wouldn't be the 1st incompatibility introduced by excessive ROM deviation from stock](https://github.com/klausw/hackerskeyboard/issues/535).

**Comment by jimklimov** [comment]
Yes, I retried and it (mis-)behaves the same. Thanks for confirming this "should" have worked at least :)
